File: microservices/log-service/api/dtos/request_log.go
package dtos

type RequestLogDto struct {
	Timestamp      string `json:"timestamp"`
	Ip             string `json:"ip"`
	Url            string `json:"url"`
	UserAgent      string `json:"userAgent"`
	RequestString  string `json:"requestString"`
	StatusCode     int    `json:"statusCode"`
	ResponseString string `json:"responseString"`
}

File: ScienceArchive.Server/src/ScienceArchive.Shared.Abstractions/Logging/Models/RequestLog.cs
namespace ScienceArchive.Shared.Abstractions.Logging.Models;

public record RequestLog
{
	public DateTime Timestamp { get; set; }
	public required string Ip { get; set; }
	public required string Url { get; set; }
	public required string UserAgent { get; set; }
	public string? RequestString { get; set; }
	public string? ResponseString { get; set; }
}